apps/etterlatte-saksbehandling-ui/client/src/components/behandling/trygdetid/TrygdetidGrunnlag.tsx

```diff
@@ -56,7 +58,7 @@ export const TrygdetidGrunnlag = ({
       {
         behandlingId: behandlingId,
         // Flippe verdi av prorata for å matche PESYS
```

Comment:
Tror kanskje ikke vi trenger å referere til Pesys her lengre. Det er vel strengt talt for å matche vår egen backend.